recipes/default.rb

```diff
@@ -54,6 +54,6 @@
     action :nothing
     source 'sysctl.conf.erb'
     mode '0644'
-    notifies :start, 'service[procps]', :immediately
+    notifies :restart, 'service[procps]', :immediately
```

Comment:
I do not think this works for Ubuntu. At least on my Trusty (14.04) system,
`restart procps` fails with `restart: Unknown instance:`. `service procps restart` does seem to work however.

So I am open to this change, but we need to make sure it is backwards compatible.